src/components/PricingTable/Unsubscribe.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Button, Modal } from 'antd';
+import get from 'lodash/get';
+import * as typeformEmbed from '@typeform/embed';
+import { deleteAppSubscription, getAppPlan } from '../../batteries/modules/actions';
+import { displayErrors } from '../../utils/helper';
+
+const TYPE_FORM_UNLOADED = 0;
+const TYPE_FORM_LOADED = 1;
+const TYPE_FORM_SUBMITTED = 2;
```

Comment:
Add them on constant files, its easier to find and we can extend them to be used on different places in future of we want.